eng/scripts/Language-Settings.ps1

```diff
@@ -3,6 +3,36 @@ $LanguageDisplayName = "Swift"
 $PackageRepository = "CocoaPods"
 $packagePattern = "*.podspec.json"
 
+function Get-AllPackageInfoFromRepo($serviceDirectory)
+{
+    $allPackageProps = @()
+    $searchPath = "sdk"
+
+    if ($serviceDirectory)
+    {
+        $searchPath = Join-Path sdk ${serviceDirectory}
+    }
```

Comment:
`Join-Path` handles an empty variable (e.g. `Join-Path "/repo/sdk" $null`, so I think this can be simplified:

```suggestion
    $searchPath = Join-Path "sdk" $serviceDirectory
```